code/modules/clothing/under/miscellaneous_vr.dm

```diff
@@ -127,9 +127,12 @@
 	var/emagged = FALSE
 	var/target_size = 1
 
-/obj/item/clothing/gloves/bluespace/mob_can_equip(mob/M, gloves, disable_warning = 0)
-	. = ..()
-	if(. && ishuman(M) && !disable_warning)
+/obj/item/proc/equip_special()
```

Comment:
The PR as a whole looks good. Although I do think this 'base' proc could use being defined in some more generalized place rather than hidden away here.